openaps_reports.py

```diff
@@ -98,3 +105,12 @@ def recent_dose(self):
                 return self._read_json(Settings.SET_DOSE, [])
 
         return []
+
+    def data_age(self):
+        fn = self._get_report_path(Settings.PREDICT_GLUCOSE)
+
+        try:
+            last_update = datetime.fromtimestamp(os.path.getmtime(fn))
+            return (datetime.now() - last_update).total_seconds()
+        except:
```

Comment:
Can you be explicit about the expected error? I'm assuming the only possible exception here is `os.error` which would be if the `PREDICT_GLUCOSE` file doesn't exist.